Zend/tests/enum/011.phpt

```diff
@@ -0,0 +1,16 @@
+--TEST--
+Enum is abstract
+--FILE--
+<?php
+
+enum Foo {}
+
+try {
+    new Foo();
```

Comment:
Could also test that `new Foo::Bar()` is/isn't allowed?


> match expressions offer a natural and convenient way to branch logic depending on the enum value. **Since every instance of a Unit Case is a singleton,** it will always pass an identity check.

So would it be forbidden when the implementation is finished?

Reply:
> So would it be forbidden when the implementation is finished?

Exactly. This is simply not implemented yet.

Reply:
This works fine now.